sunflower-land/sunflower-land
# [FIX] Visit farm button fix

# Description

This PR covers the ff:
- `Enter` key now loads the visiting farm
- ~~Counter now excludes locked crops~~
- ~~Counter now ignores `readonly/visit` mode~~
- Fixed inventory check for zone four in `visitUtils`

Fixes #issue N/A

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Manual testing
~~INITIAL_FARM with Pumpkin Soup~~
Steps:
1. Load your farm -- counter should be correct and excludes locked crops~~


Visit Farm
Steps
1. Sign in metamask
2. Explore a friend's farm
3. input farm ID
4. press `Enter` key -- should load the visiting farm

# Checklist:
